add personal bio page (Batch 13) Issue B

kastentx opened the pull request:
## Description

Adds a personal bio page for Nick Kasten _(see screenshot below)_:
<img width="1280" alt="Screenshot 2025-02-19 at 11 18 58 PM" src="https://github.com/user-attachments/assets/6d3bdb5a-6ae4-477f-ab28-97ea61d02e39" />

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #8_

Your ENS/address: 0x0c4d0953b769637de90cc3fe42f6f356543c5a45

melanke commented on packages/nextjs/app/builders/0x0C4D0953B769637DE90CC3Fe42f6f356543C5a45/page.tsx:
```diff
@@ -0,0 +1,49 @@
+      </div>
+
+      <div className="mb-6">
+        <pre className="font-semibold">0x0c4d0953b769637de90cc3fe42f6f356543c5a45</pre>
```
There's an SE2 component called `<Address>` that might fit well to present this information.

melanke commented on packages/nextjs/app/builders/0x0C4D0953B769637DE90CC3Fe42f6f356543C5a45/page.tsx:
```diff
@@ -0,0 +1,49 @@
+
+      <div className="mb-4">
+        <h2 className="text-xl font-semibold text-gray-700 dark:text-gray-200">Contact</h2>
+        <p className="text-gray-600 dark:text-gray-300">📍 Austin TX, USA</p>
+        <p className="text-gray-600 dark:text-gray-300">📧 kastencode@gmail.com</p>
```
There are special HTML5 tags that fit well for this kind of information.

melanke reviewed (changes requested):
Thanks @kastentx for your personal page!  

You have very good code here, but I will be very picky in this review so we can have a little bit of interaction ^^

kastentx commented:
@melanke Thanks for the review and the helpful comments. The heads up on the SE2 component was especially cool! I implemented the requested changes and also went ahead and added a link for Telegram.

<img width="849" alt="Screenshot 2025-02-20 at 3 53 32 PM" src="https://github.com/user-attachments/assets/fdf74687-ebb7-49ab-9e73-fc7f6bf509b1" />

melanke commented:
<img width="257" alt="image" src="https://github.com/user-attachments/assets/45121619-3d7c-4955-a542-b27fcc3bee92" />

I think I have OCD: if it has a rounded border, it can't touch anything else 😅

kastentx commented:
@melanke Good catch! I removed rounding from the top edge and I think it looks a lot cleaner:

<img width="1086" alt="Screenshot 2025-02-20 at 7 29 31 PM" src="https://github.com/user-attachments/assets/ab140b6d-42da-45dc-af97-7b0e8473a81d" />

phipsae commented:
Thanks for your personal page! :) Lets merge!
